os_stub/cryptlib_openssl/pk/x509.c

```diff
@@ -1887,6 +1887,7 @@ bool libspdm_x509_verify_cert(const uint8_t *cert, size_t cert_size,
      * still trusted intermediate certificate. Also disable time checks.*/
 
     X509_STORE_set_flags(cert_store, X509_V_FLAG_PARTIAL_CHAIN);
+    X509_STORE_set_flags(cert_store, X509_V_FLAG_NO_CHECK_TIME);
```

Comment:
Can we just fix comment?

I forget why it is comment in this way.

Reply:
A lot of SPDM devices don't have reliable clocks, so I thought skipping the time check made a lot of sense

Reply:
Then, this is a functional change.
Please file an issue and let's discuss it in SPDM TF meeting.

Reply:
It's not really a functional change, the comment already says we do this. I have created an issue here: https://github.com/DMTF/libspdm/issues/2237

Reply:
If code does not have the check, then it is a functional change.

You can treat it as a comment bug.